File: Problem Set 1 - Intro to C/Mario.c
#include <cs50.h>
#include <stdio.h>
 
int main(void)
{
   int n;
   do
   {
       n = get_int("Height: ");
   }
   while (n < 1 || n > 8);
 
   for (int i = 0; i < n; i++)
   {
    // for each element less than n - i - 1
       for (int j = 0; j < n - 1 - i; j++)
       {
           printf(" ");
       }
       for (int j = 0; j < i + 1; j++)
       {
           printf("#");
       }
       printf("\n");
   }
 
 
  
  
  
}
 

File: Problem Set 6 - Python/mario/more/mario.py
from cs50 import get_int

n = get_int("Height:    ")

while (n < 1 or n > 8):
    n = get_int("Height:    ")

for i in range(n):
    print(' ' * (n-i-1), end="")
    print('#' * (i+1), end="")
    print('  ', end="")
    print('#' * (i+1))
